### PR TITLE
[update] use explicit argument names

### PR DESCRIPTION
**Pull request by georgehristov**
Simple change name of arguments

### CODE REVIEW

**Comment by mvorisek** [comment]
There are still a lot of other similar names (search for `\$m(?!\w)`, even as function params).

I think `$m` is ok as long as it is not used for anything else. But if it will be normalized everywhere, I think `$model` can be ok, but I would still prefer `$m` in anonymous functions etc.

What is the rule?

**Comment by georgehristov** [comment]
We all are setting the rules but using explicit names makes code clear to read for newcomers and also for cooperating or returning to your own code.
I prefer avoiding single letter vars unless `$k=>$v` in some small loop.

**Comment by mvorisek** [comment]
> I prefer avoiding single letter vars unless `$k => $v` in some small loop.

I agree. I think we should follow about this, but then it needs to be really consistent, not 50 PRs to fix something that should be fixed in one.

The same for `$f` which is even more ambiguous as used for `Form` and `Field`, maybe even for some `Control` still...

So at least all these should be fixed in this PR.

**Comment by georgehristov** [comment]
Okay, I will search for all and add in this PR

**Comment by PhilippGrashoff** [comment]
nice PR you're working on. I agree on not liking the 1 letter var names. $model is so much more meaningful than $m, as is $field instead of $f.

File: src/View.php [hunk 1]
```diff
@@ -215,11 +215,9 @@ public function __construct($label = null, $class = null)
      *
      * @return Model
      */
-    public function setModel(Model $m)
+    public function setModel(Model $model)
     {
-        $this->model = $m;
-
-        return $m;
+        return $this->model = $model;
```
Comment (mvorisek) [inline src/View.php]: I think 2 lines are much better to read - you assign and then you return.

File: demos/_includes/Persistence_Faker.php [hunk 2]
```diff
@@ -60,8 +60,8 @@ public function export($m, $fields = [])
             $data[] = $row;
         }
 
-        return array_map(function ($r) use ($m) {
-            return $this->typecastLoadRow($m, $r);
+        return array_map(function ($r) use ($model) {
```
Comment (mvorisek) [inline demos/_includes/Persistence_Faker.php]: maybe `$row`

File: demos/_includes/SomeData.php [hunk 3]
```diff
@@ -8,21 +8,21 @@ class SomeData extends \atk4\data\Model
 {
     public function __construct()
     {
-        $p = new Persistence_Faker();
+        $persistence = new Persistence_Faker();
```
Comment (mvorisek) [inline demos/_includes/SomeData.php]: `$fakerPersistence`

File: src/View.php (hunk 1 above)
Comment (georgehristov) [reply src/View.php]: reverted

File: demos/interactive/wizard.php [hunk 4]
```diff
@@ -27,38 +27,38 @@
...
 // and set a custom js action or even set a different link. You can use recall()
 // to access some values that were recorded on another steps.
-$t->addStep(['Select Model', 'description' => '"Country" or "Stat"', 'icon' => 'table'], function (Wizard $w) {
+$wizard->addStep(['Select Model', 'description' => '"Country" or "Stat"', 'icon' => 'table'], function (Wizard $w) {
     if (isset($_GET['name'])) {
         $w->memorize('model', $_GET['name']);
         $w->app->redirect($w->urlNext());
     }
 
     $c = \atk4\ui\Columns::addTo($w);
```
Comment (mvorisek) [inline demos/interactive/wizard.php]: ideally fix all one char variables related with components like this

File: demos/interactive/wizard.php [hunk 5]
```diff
@@ -27,38 +27,38 @@
...
 
-    $t = \atk4\ui\Grid::addTo($c->addColumn(), ['paginator' => false, 'menu' => false]);
+    $wizard = \atk4\ui\Grid::addTo($c->addColumn(), ['paginator' => false, 'menu' => false]);
```
Comment (mvorisek) [inline demos/interactive/wizard.php]: hmm, it is Grid... Wizar is defined above (line 46) and not renamed

File: demos/interactive/wizard.php (hunk 4 above)
Comment (georgehristov) [reply demos/interactive/wizard.php]: fixed

File: demos/interactive/wizard.php (hunk 5 above)
Comment (georgehristov) [reply demos/interactive/wizard.php]: fixed

File: demos/_includes/Persistence_Faker.php (hunk 2 above)
Comment (georgehristov) [reply demos/_includes/Persistence_Faker.php]: fixed

File: demos/_includes/SomeData.php (hunk 3 above)
Comment (georgehristov) [reply demos/_includes/SomeData.php]: fixed

**Comment by mvorisek** [comment]
Still more names to refactor, example:
image

**Review by mvorisek — APPROVED** [review]
LGTM, but more renames can be done to make this PR really complete one

**Comment by mvorisek** [comment]
All snake_case variable names should be fixed to camelCase - regex `(?<! (public|protected|private) )\$\w+_\w+`

**Comment by georgehristov** [comment]
We can work on it further but let's merge this now to avoid conflicts.

**Comment by mvorisek** [comment]
already approved, but I think at least another similar usecases in demos should be fixed

**Pull request merged**